drone.rb

```diff
@@ -17,29 +28,23 @@ class Drone < Formula
 
   def install
     if build.head?
-      drone_build_home = "/tmp/#{SecureRandom.hex}"
-      drone_build_path = File.join(drone_build_home, "src", "github.com", "drone", "drone-cli")
-
-      ENV["GOPATH"] = drone_build_home
-      ENV["GOHOME"] = drone_build_home
-
-      mkdir_p drone_build_path
-
-      system("cp -R #{buildpath}/* #{drone_build_path}")
-      ln_s File.join(cached_download, '.git'), File.join(drone_build_path, '.git')
+      mkdir_p buildpath/File.join("src", "github.com", "drone")
+      ln_s buildpath, buildpath/File.join("src", "github.com", "drone", "drone")
 
-      Dir.chdir drone_build_path
+      ENV["DRONE_BUILD_NUMBER"] = "homebrew"
+      ENV["GOVENDOREXPERIMENT"] = "1"
+      ENV["GOPATH"] = buildpath
+      ENV["GOHOME"] = buildpath
+      ENV["PATH"] += ":" + File.join(buildpath, "bin")
 
-      system("go get ./...")
-      system("go build -o ./drone_cli github.com/drone/drone-cli/drone")
+      system("make deps_backend")
```

Comment:
For some reason, using just `make deps` didn't preserve order (`deps_backend` before `deps_frontend`), and I had issues with `go-bindata` not being installed prior to when it was needed. This explicit call execution of `deps_backend` first solves the issue.

(Wonder if this could be an issue w/BSD make versus GNU?)